CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - **Breaking change** The api now uses the module name instead of the prefix to generate the storage methods. The methods of the grandpa module changed from `api.query.grandpaFinality` to `api.query.grandpa`. 
 - Update with latest substrate 2.x types
 - **Breaking Change** StorageFunction has been renamed to StorageEntry.
+- **Breaking Change** Vote interface extends U8a instead of I8.
```

Comment:
Why is this marked breaking? The users of the interface shouldn’t care what happens under the hood. (And if indeed breaking from an API user POV, the comment doesn’t say what to do to fix it)

I would assume the entry would be non-breaking, just supports substrate 2.x interfaces for voting

Reply:
Since I8 is a BN, all BN methods that users might be using are gone